src/app/account.service.ts

```diff
@@ -74,6 +83,54 @@ export class AccountService {
     }
   }
 
+  getDerivedUser(publicKey: string, blockHeight: number): DerivedUserInfo{
```

Comment:
nit: Rename to getDerivedPrivateUser. Also rename DerivedUserInfo to DerivedPrivateUserInfo